src/Nethermind/Nethermind.Core/Encoding/Rlp.cs

```diff
@@ -2161,5 +2161,25 @@ public static int LengthOf(Span<byte> array)
 
             return LengthOfLength(array.Length) + 1 + array.Length;
         }
+
+        public static int LengthOf(string value)
+        {
+            return LengthOf(System.Text.Encoding.ASCII.GetBytes(value));
+        }
+
+        public static int LengthOf(byte value)
+        {
+            return 1;
+        }
+        
+        public static int LengthOf(LogEntry item) 
+        {
+            if (Decoders.ContainsKey(typeof(LogEntry)))
+            {
+                ((IRlpDecoder<LogEntry>) Decoders[typeof(LogEntry)]).GetLength(item, RlpBehaviors.None);
```

Comment:
I guess you would like to return from here instead of always throwing exception
(additional comment - this needs full testing in ReceiptDecoderTests)